### PR TITLE
magento/devdocs#: Add addVirtualProductsToCart mutation API errors

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) adds information about API errors of `addVirtualProductsToCart` mutation.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/graphql/mutations/add-virtual-products.html

## Links to Magento source code

- [Could not find a cart with ID "XXX"](https://github.com/magento/graphql-ce/blob/2.3-develop/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/AddVirtualProductToCartTest.php#L211https://github.com/magento/graphql-ce/blob/2.3-develop/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/AddVirtualProductToCartTest.php#L211)
- [Could not find a product with SKU "YYY"](https://github.com/magento/graphql-ce/blob/2.3-develop/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/AddVirtualProductToCartTest.php#L211https://github.com/magento/graphql-ce/blob/2.3-develop/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/AddVirtualProductToCartTest.php#L211)
- [Required parameter "cart_id" is missing](https://github.com/magento/graphql-ce/blob/2.3-develop/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/AddVirtualProductToCartTest.php#L211https://github.com/magento/graphql-ce/blob/2.3-develop/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/AddVirtualProductToCartTest.php#L211)
- [Required parameter "cart_items" is missing](https://github.com/magento/graphql-ce/blob/2.3-develop/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/AddVirtualProductToCartTest.php#L211https://github.com/magento/graphql-ce/blob/2.3-develop/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/AddVirtualProductToCartTest.php#L211)
- [The current user cannot perform operations on cart XXX](https://github.com/magento/graphql-ce/blob/2.3-develop/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/AddVirtualProductToCartTest.php#L211)
- [The product's required option(s) weren't entered. Make sure the options are entered and try again.](https://github.com/magento/graphql-ce/blob/2.3-develop/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/AddVirtualProductWithCustomOptionsToCartTest.php#L102)

Thank you!

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/graphql/mutations/add-virtual-products.md [hunk 1]
```diff
@@ -127,3 +127,14 @@ Attribute |  Data Type | Description
 {% include graphql/cart-object.md %}
 
 [Cart query output]({{page.baseurl}}/graphql/queries/cart.html#cart-output) provides more information about the `Cart` object.
+
+## Errors
+
+Error | Description
+--- | ---
+`Could not find a cart with ID "XXX"` | The specified `cart_id` value does not exist in the `quote_id_mask` table.
+`Could not find a product with SKU "YYY"` | A simple product with the SKU specified in the `data`.`sku` attribute does not exist.
```
Comment (keharper) [inline guides/v2.3/graphql/mutations/add-virtual-products.md]: @atwixfirster Should this say "A virtual product"?

Comment (atwixfirster) [reply guides/v2.3/graphql/mutations/add-virtual-products.md]: should....

**fixed**

Thank you, @keharper

**Comment by dobooth** [comment]
running tests

**Pull request merged**